docs/api-docs/channels/building-channel-apps.md

```diff
@@ -252,26 +269,27 @@ Accept: application/json
 
 <!-- theme: info -->
 
-### Note
-> * If the external channel does not have a concept of customers, an order can be created with a `customer_id` of `0` (this value is used for "guest" shoppers).
+> **Note**
+>
+> - If the external channel does not have a concept of customers, an order can be created with a `customer_id` of `0` (this value is used for "guest" shoppers).
 
 </div>
 </div>
 </div>
 
 ## Managing orders and inventory
 
-In order to ensure that BigCommerce merchants are able to continue using their existing catalog and eCommerce workflows, in addition to enabling other downstream eCommerce functionality, we require channel apps sync order and inventory information back to BigCommerce and ensure the following use cases are supported:
-* Creating orders as they are made on the external channel
-* Updating orders as changes are made on the external channel
-* Reading orders, via API or webhooks, to push any necessary changes / updates made by the merchant in the BigCommerce control panel to the external channel
-* Reading inventory levels, via the products API or webhooks, to get up-to-date inventory levels for the channel. This is critical, as orders can come from other channels including their main storefront that impact inventory available to the channels your app supports.
+To ensure that BigCommerce merchants are able to continue using their existing catalog and eCommerce workflows, in addition to enabling other downstream eCommerce functionality, we require channel apps sync order and inventory information back to BigCommerce and ensure the following use cases are supported:
 
-In order to do this, integrate the following endpoints:
+- Creating orders as they are made on the external channel
+- Updating orders as changes are made on the external channel
+- Reading orders, via API or webhooks, to push any necessary changes/updates made by the merchant in the BigCommerce control panel to the external channel
+- Reading inventory levels via the products API or webhooks to get up-to-date inventory levels for the channel. This is critical, as orders can come from other channels, including their main storefront that impact inventory available to the channels your app supports.
```

Comment:
I think we should re-write this section to make it more concise. 
```suggestion
- Reading inventory levels via the Products API or webhooks to get up-to-date inventory levels for the channel. This is critical, because orders that impact available inventory can come from other channels including the main storefront. 

```